File: src/test/resources/output/ruby/nethttp/headers.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Get.new(url)
request["x-foo"] = "Bar"
request["accept"] = "application/json"

response = http.request(request)
puts response.read_body

File: src/test/resources/output/java/okhttp/headers.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
  .url("http://mockbin.com/har")
  .get()
  .addHeader("x-foo", "Bar")
  .addHeader("accept", "application/json")
  .build();

Response response = client.newCall(request).execute();
